packages/master/master.js

```diff
@@ -1943,6 +1943,7 @@ async function startServer() {
 	// Make sure we're actually going to listen on a port
 	let httpPort = masterConfig.get("master.http_port");
 	let httpsPort = masterConfig.get("master.https_port");
+	let bindIp = masterConfig.get("master.address_bind");
```

Comment:
If you add `|| ""` after the `.get()` it becomes an empty string when null and you don't need the if statements further down. The variable should also be named the same as the option (like the ones above it).

Reply:
okey